tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -234,20 +259,49 @@ def test_extension_is_not_included(self) -> None:
         ):
             # No extensions are supported, thus, we can't generate a code mapping
             self._process_and_assert_no_code_mapping(
-                repo_files=[file_in_repo],
+                repo_trees={REPO1: [file_in_repo]},
                 frames=[self.frame(frame_filename, True)],
                 platform=platform,
             )
 
             with patch(f"{REPO_TREES_CODE}.get_supported_extensions", return_value=["tbd"]):
                 self._process_and_assert_code_mapping(
-                    repo_files=[file_in_repo],
+                    repo_trees={REPO1: [file_in_repo]},
                     frames=[self.frame(frame_filename, True)],
                     platform=platform,
                     expected_stack_root="foo/",
                     expected_source_root="src/foo/",
                 )
 
+    def test_multiple_repositories(self) -> None:
+        platform = "foo"
+        repo_trees = {
+            REPO1: ["src/foo/bar.py"],
+            REPO2: ["app/baz/qux.py"],
+        }
+        with (
+            patch(f"{CODE_ROOT}.utils.get_platform_config", return_value={}),
+            patch(f"{REPO_TREES_CODE}.get_supported_extensions", return_value=[]),
+        ):
+            self._process_and_assert_code_mapping(
+                repo_trees=repo_trees,
+                frames=[self.frame("foo/bar.py", True)],
+                platform=platform,
+                expected_stack_root="foo/",
+                expected_source_root="src/foo/",
+                expected_repo_name=REPO1,
+                expected_num_code_mappings=1,
+            )
+            self._process_and_assert_code_mapping(
+                repo_trees=repo_trees,
+                frames=[self.frame("baz/qux.py", True)],
+                platform=platform,
+                expected_stack_root="baz/",
+                expected_source_root="app/baz/",
+                expected_repo_name=REPO2,
+                expected_num_code_mappings=2,
```

Comment:
Checking that the code mapping is associated to REPO2 and that we now have two code mappings.